### ISSUE TITLE
View all editions maxes out at 1000

### ISSUE THREAD

**Issue by seabelis** [internetarchive/openlibrary#7233]
Some works, generally those in the public domain, have over 1000 edition records. Clicking the view all links limits the edition count to 1000 records, making the rest unaccessible. This is inconsistent; sometimes all editions will load.
### Evidence / Screenshot (if possible)
<img width="1299" alt="Screenshot 2022-12-07 at 12 16 53" src="https://user-images.githubusercontent.com/17739465/206165470-a55a5f7c-066d-43eb-856e-2a3bacb4e240.png">

### Relevant url?

https://openlibrary.org/works/OL17732W/Bible?mode=all

### Steps to Reproduce

1. Go to ...a work of 1000+ editions
2. Do ... view all

* Actual: sometimes all editions load, sometimes the max is 1000
* Expected: all editions should load all of the time

### Details

- **Logged in (Y/N)?**
- **Browser type/version?**
- **Operating system?**
- **Environment (prod/dev/local)?** prod

### Stakeholders

@mekarpeles

**Comment by mekarpeles** [comment]
Let's try removing the limit and if we see sentry errors or performance issues, we'll followup.

**Comment by seabelis** [comment]
Yes. I use the filter frequently to search for a specific year, publisher, ISBN, language, etc. If I want to avoid adding a duplicate, I need to see what we already have.

**Comment by cdrini** [comment]
Ah ok, so if we had the search box search over _all_ editions, you don't need 2000 rendered at once. That's useful context! With edition aware search we should be able to search all the editions.

**Comment by seabelis** [comment]
The way the filter on the editions table has been working is fine. It just needs to access all available editions. This works normally, just not when solr updater is laggy.
